automatic/youtube-dl/update.ps1

```diff
@@ -0,0 +1,41 @@
+import-module au
+
+$releases = 'https://api.github.com/repos/rg3/youtube-dl/releases/latest'
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(^[$]url\s*=\s*)('.*')"      = "`$1'$($Latest.URL32)'"
+      "(^[$]checksum\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+      "(^[$]checksumType\s*=\s*)('.*')" = "`$1'$($Latest.ChecksumType32)'"
+        }
+    }
+}
+
+function global:au_GetLatest {
+  $headers = @{}
+  if (Test-Path Env:\github_api_key) {

```

Comment:
What actually happens here if no env var is present ?

I wanted to use Github API before but in my case releases page actually had more releases then what API returns (pre releases AFAIR).